packages/core/src/tracing/instrumentation/database/couchbase.js

```diff
@@ -297,6 +308,48 @@ function instrumentQueryIndexes(cluster, connectionStr) {
   };
 }
 
+function instrumentAnalyticsIndexes(cluster, connectionStr) {
+  const origAnalyticsIndex = cluster.analyticsIndexes;
+
+  cluster.analyticsIndexes = function instanaAnalyticsIndexes() {
+    const analyticsIndexes = origAnalyticsIndex.apply(this, arguments);
+
+    [
+      'createIndex',
+      'getAllIndexes',
+      'dropIndex',
+      'createDataverse',
+      'dropDataverse',
+      'createDataset',
+      'getAllDatasets',
+      'dropDataset'
+    ].forEach(fnName => {
+      shimmer.wrap(analyticsIndexes, fnName, function instanaInstrumentOperationWrapped(original) {
+        return function instanaInstrumentOperationWrappedInner() {
+          const originalThis = this;
+          const originalArgs = arguments;
+
+          return instrumentOperation(
+            {
+              connectionStr,
+              sql: camelCaseToUpperWords(fnName),
+              resultHandler: (span, result) => {
+                if (result) {
+                  span.data.couchbase.bucket = result[0].BucketName;
```

Comment:
Fixed the test, we can use result[0].bucketName because we get an Array result and the object is directly accessible using the index value.
executing `result[0].bucketName` will get us `projects`.

verified it and added test cases

Reply:
Oki sweet